source/Halibut/Transport/Protocol/MessageExchangeStream.cs

```diff
@@ -156,8 +159,7 @@ public void IdentifyAsSubscriber(string subscriptionId)
 
         public void IdentifyAsServer()
         {
-            streamWriter.Write("MX-SERVER ");
-            streamWriter.Write(currentVersion.ToString());
+            streamWriter.Write($"{MxServer} {currentVersion}");
```

Comment:
I wonder if we should make this a WriteLine instead?

Reply:
Will do this in the next PR.